app/src/main/java/org/pesmypetcare/mypetcare/services/MealManagerService.java

```diff
@@ -31,28 +32,28 @@
      * @param newDate The new date of the meal
      * @param oldDate The old date of the meal
      */
-    void updateMealDate(User user, Pet pet, String newDate, String oldDate);
+    void updateMealKey(User user, Pet pet, String newDate, String oldDate) throws ExecutionException, InterruptedException;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 123).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5527577)